app/services/group/seed.rb

```diff
@@ -44,6 +51,28 @@ def mark_as_seeded
       RbacSeed.find_or_create_by(:external_tenant => @user.tenant) if @seeded.data.present?
     end
 
+    def add_user_to_catalog_admin_group
+      group_principal_in = RBACApiClient::GroupPrincipalIn.new.tap do |group|
+        group.principals = [RBACApiClient::PrincipalIn.new(:username => @user.username)]
+      end
+
+      # call the group api again to get a list of all the groups that were created
+      lookup_groups
+
+      RBAC::Service.call(RBACApiClient::GroupApi) do |api|
+        api.add_principal_to_group(get_group_uuid("Catalog Administrators"), group_principal_in)
```

Comment:
Are we listing all the groups that currently exist?  If other Services have created groups for their own purposes would we hit a paging issue?  Can we just look up the group by name?

And the `Catalog Administrators` value should be moved to a constant.

Reply:
Changed it to a `RBAC::Paginate` call while specifying `:name => CATALOG_ADMINISTRATOR_GROUP_NAME` so it searches by name.